subsys/bluetooth/mesh/shell.c

```diff
@@ -1206,6 +1237,48 @@ static int cmd_mod_app_bind(const struct shell *shell, size_t argc,
 	return 0;
 }
 
+
+static int cmd_mod_app_unbind(const struct shell *shell, size_t argc,
+			    char *argv[])
+{
+	uint16_t elem_addr, mod_app_idx, mod_id, cid;
+	uint8_t status;
+	int err;
+
+	if (argc < 4) {
+		return -EINVAL;
+	}
+
+	elem_addr = strtoul(argv[1], NULL, 0);
+	mod_app_idx = strtoul(argv[2], NULL, 0);
+	mod_id = strtoul(argv[3], NULL, 0);
+
+	if (argc > 4) {
+		cid = strtoul(argv[4], NULL, 0);
+		err = bt_mesh_cfg_mod_app_unbind_vnd(net.net_idx, net.dst,
+						   elem_addr, mod_app_idx,
+						   mod_id, cid, &status);
+	} else {
+		err = bt_mesh_cfg_mod_app_unbind(net.net_idx, net.dst, elem_addr,
+					       mod_app_idx, mod_id, &status);
+	}
+
+	if (err) {
+		shell_error(shell, "Unable to send Model App Bind (err %d)",
+			    err);
+		return 0;
+	}
+
+	if (status) {
+		shell_print(shell, "Model App Bind failed with status 0x%02x",
+			    status);
+	} else {
+		shell_print(shell, "AppKey successfully bound");
+	}
+
+	return 0;
+}
+
```

Comment:
Please add these two to the command list at the bottom, and add corresponding documentation entries in doc/reference/bluetooth/mesh/shell.rst :)